schema.yaml

```diff
@@ -53,38 +58,65 @@ properties:
     title: Gateway Hostname
     description: The hostname for your Gateway instance
     type: string
+    title: Gateway Hostname
```

Comment:
i think a few of these need deduplication - i see there's already a title/description for some of these adds.

Reply:
Merge conflicts I missed, thanks!